source/audio/midiplayer.cpp

```diff
@@ -279,3 +297,26 @@ bool MidiPlayer::isPlaying() const
 {
     return myIsPlaying;
 }
+
+int MidiPlayer::getPlayerFromChannel(const int channel)
```

Comment:
An alternative to this function would be to record the player number on the `MidiEvent`, which avoids having to duplicate the logic for how players are assigned to channels. (There's actually an unused `myPlayer` member variable in `MidiEvent` that was intended for this.purpose, but the `myInstrument` member should probably be removed).

What are your thoughts on that?

If we stick with the current approach, those unused members should be removed, and there's also a compiler warning (below) where the compiler incorrectly thinks the result could be uninitialized:
```
../source/audio/midiplayer.cpp:314:14: warning: variable 'player' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]
    else if (channel > METRONOME_CHANNEL)
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~
../source/audio/midiplayer.cpp:319:12: note: uninitialized use occurs here
    return player;
           ^~~~~~
../source/audio/midiplayer.cpp:314:10: note: remove the 'if' if its condition is always true
    else if (channel > METRONOME_CHANNEL)
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../source/audio/midiplayer.cpp:303:15: note: initialize the variable 'player' to silence this warning
    int player;
              ^
               = 0
1 warning generated.
```